doc/nrf/ug_thread_ot_memory.rst

```diff
@@ -0,0 +1,90 @@
+.. _thread_ot_memory:
+
+OpenThread memory requirements
+##############################
+
+This page provides information about the layout and the amount of Flash and RAM memory required by the sample using the OpenThread stack.
+These values are useful to see whether your application has enough space for running on paricular platforms in different OpenThread configurations.
```

Comment:
```suggestion
These values are useful to see whether your application has enough space for running on particular platforms in different OpenThread configurations.
```
What platforms and configurations are we talking about here? You mean different boards and feature sets? If so, link to supported boards page and feature set section, please.